data/language/nl-NL.yml

```diff
@@ -2191,3 +2191,209 @@ strings:
   2136: "{COLOUR WINDOW_2}Zoomen op aanwijzerpositie"
   2137: "{SMALLFONT}{COLOUR BLACK}Als dit ingeschakeld, zal er worden ingezoomd op de locatie van de aanwijzer, in plaats van in het midden van het scherm."
   2138: "{COLOUR WINDOW_2}Muziek afspelen op titelscherm"
+  2139: "Terug naar menu"
+  2140: "OpenLoco afsluiten"
+  2141: "{COLOUR WINDOW_2}Schakel AI-bedrijven uit"
+  2142: "{SMALLFONT}{COLOUR BLACK}Dit schakelt het 'nadenken' van AI-bedrijven uit, wat ze direct stillegt.{NEWLINE}In nieuwe spellen voorkomt het ook dat AI-bedrijven worden opgezet."
+  2143: "{COLOUR WINDOW_2}Hoeveelheid autosaves:"
+  2144: "{COLOUR WINDOW_2}Frequentie van autosaves:"
+  2145: "Nooit"
+  2146: "Elke maand"
+  2147: "Elke {INT32} maanden"
+  2148: "{COLOUR WINDOW_2}Bron van hoogtemap:"
+  2149: Originele generator
+  2150: Simplex-generator
+  2151: "Voertuig aanpassen"
+  2152: "Kloon voertuig"
+  2153: "Kan dit voertuig niet klonen..."
+  2154: "Zoek voertuig op in hoofdvenster"
+  2155: "Volg voertuig in hoofdvenster"
+  2156: "Schakel het cheats/debug-toolbarmenu in"
+  2157: "{SMALLFONT}{COLOUR BLACK}Dit voegt een extra knop toe aan de toolbar van het spel die toegang geeft tot cheats en debugopties."
+  2158: "Schakel zandbak-modus in"
+  2159: "Allow manual driving"
+  2160: "Allow building while paused"
+  2161: "Geef fps-teller weer"
+  2162: "{SMALLFONT}{COLOUR BLACK}Dit toont een teller aan de bovenkant van het scherm die aangeeft hoeveel frames er per seconde worden gerenderd."
+  2163: "Framebeperking opheffen"
+  2164: "{SMALLFONT}{COLOUR BLACK}Dit heft de beperking rond renderen op, wat leidt tot een verversingsfrequentie van meer dan 40Hz."
+  2165: "Hardware"
+  2166: "Map rendering"
+  2167: "Tegelinspecteur"
```

Comment:
Naar mijn idee is het woord "vakje" gangbaarder dan "tegel".

Reply:
Ik vind zelf "vakje" toch wat minder duidelijk. Voor mij suggereert dat iets om wat _in_ te doen, in plaats van _op_ te zetten. Ik laat het even zo.